test/SlowTests/Issues/RavenDB_23091.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations.TimeSeries;
+using SlowTests.Core.Utils.Entities;
+using Sparrow;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23091 : RavenTestBase
+    {
+        public RavenDB_23091(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.TimeSeries)]
+        public async Task ShouldRemoveTimeSeriesAfterRetention()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var raw = new RawTimeSeriesPolicy(TimeValue.FromHours(24));
+
+                var config = new TimeSeriesConfiguration
+                {
+                    Collections = new Dictionary<string, TimeSeriesCollectionConfiguration>
+                    {
+                        ["Users"] = new TimeSeriesCollectionConfiguration
+                        {
+                            RawPolicy = raw
+                        }
+                    },
+                    PolicyCheckFrequency = TimeSpan.FromSeconds(1)
```

Comment:
just to make the test cleaner, you can remove this, so you will not get a concurrent access to the policy runner

Reply:
done